### PR TITLE
Added basic login page

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
Title: `Added basic login page`

Description:
```
- Added basic login page
- Added styling and link to LS login, but no functionality linking post-login.
- Added button in header navigation for login page.
- closes GDEV-1131
- closes GDEV-1162
- closes GDEV-1163
- closes GDEV-1164
```

### CODE REVIEW

File: src/components/login/login.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        </a>
+      </div>
+      <hr className="bg-secondary text-center"/>
+      <div className="bg-light-alternative p-4 my-5 rounded text-center">
+        <p className="mb-4">
+          If you do not have a <strong>Life Science</strong> account, you can
+          register here.
+        </p>
+        <a href="https://signup.aai.lifescience-ri.eu/fed/registrar/?vo=lifescience">
+          <img src={lsRegister} alt="LS Register" width="200px" />
```
Comment (alamgirmani) [inline src/components/login/login.tsx:26]: The login and register links seems to open the same page although the links are different

File: src/components/login/login.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        </a>
+      </div>
+      </div>
+    </Container>
```
Comment (alamgirmani) [inline src/components/login/login.tsx:30]: The page height seems to be less the screen size and the footer is really high up.

File: src/components/login/login.tsx (hunk 1 above)
Comment (ac-jorellanaf) [reply src/components/login/login.tsx:26]: I got the links from the LS Login info page. I guess we can just have the same URL for now?

File: src/components/login/login.tsx (hunk 2 above)
Comment (ac-jorellanaf) [reply src/components/login/login.tsx:35]: I hope I improved it a bit

**Review by alamgirmani — APPROVED** [review]

**Pull request merged**